pandas/tests/io/test_excel.py

```diff
@@ -179,6 +179,42 @@ def test_usecols_str(self, ext):
         tm.assert_frame_equal(df2, df1, check_names=False)
         tm.assert_frame_equal(df3, df1, check_names=False)
 
+    def test_usecols_str_list(self, ext):
+
+        dfref = self.get_csv_refdf('test1')
+
```

Comment:
can you parameterize this test

Reply:
Hi, thank you for yur feedback! =)

I'm not sure I've got it right. Do you want me to parametrize the objects passed as argument to parse_cols argument to each inside of test_usecols_str_list test because its tests are a copy of test_usecols_str or you want me to parametrize something else?

Reply:
you are testing multiple cases that are pretty similar. the test becomes much simpler if you can parameterize over the possible cases.

Reply:
Ok, I got it. Do you want me to do the same to the test above (test_usecols_str)?

Reply:
Hi @jreback. I implemented all requested changes in the code.
I would like to know your opinion about a suggestion made by @chris-b1 at https://github.com/pandas-dev/pandas/pull/20480#issuecomment-375986768.